moscar.go

```diff
@@ -1,124 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"github.com/micromdm/squirrel/munki/datastore"
-	"github.com/micromdm/squirrel/munki/models"
-	"html/template"
-	"io"
-	"log"
-	"net/http"
-	"time"
-)
-
-const STATIC_URL string = "/static/"
-const STATIC_ROOT string = "static/"
-const REPO_PATH string = "/Users/Shared/repo"
-
-type Context struct {
-	Static string
-	Prods  []ParsedPkginfo
-}
-
-// Template expects icon, 'product' name, description, and version, which becomes the download URL
-type ParsedPkginfo struct {
-	Icon        string
-	Name        string
-	Descript    string
-	Version     string
-	DownloadURL string
-}
-
-func Parse(in_pkginfo *models.PkgsInfo, ppi *ParsedPkginfo) *ParsedPkginfo {
-	ppi.Icon = in_pkginfo.IconName
-	ppi.Name = in_pkginfo.Name
-	ppi.Descript = in_pkginfo.Description
-	ppi.Version = in_pkginfo.Version
-	ppi.DownloadURL = in_pkginfo.PackageCompleteURL
-	return ppi
-}
-
-func Home(w http.ResponseWriter, req *http.Request) {
-	repo := datastore.SimpleRepo{Path: REPO_PATH}
-	log.Print("found repo: ", repo)
-	catalog, err := repo.AllPkgsinfos()
-	if err != nil {
-		log.Fatal("faux-catalog building error: ", err)
```

Comment:
Don't Fatal, which will crash the program. Instead, return an error to the caller.
A better alternative here is to do:
```
	catalog, err := repo.AllPkgsinfos()
	if err != nil {
		http.Error(w,
			fmt.Sprintf("failed to build catalog: %q", err.Error()),
			http.StatusInternalServerError,
		)
		log.Println("faux-catalog building error: ", err)
		return
	}
```

The `http.Error` will return a proper error to the client(browser).